utils.zsh/functions/check_cross

```diff
@@ -0,0 +1,23 @@
+autoload -Uz log_info log_status log_error log_debug log_warning
+
+log_debug 'Checking for apt-get...'
+if (( ! ${+commands[apt-get]} )) {
+  log_error 'No apt command found - please install apt'
+  return 2
+} else {
+  log_debug "Apt located at ${commands[apt-get]}"
```

Comment:
Apt -> Apt-get

This casing is a bit weird, but avoiding it requires more words or just not capitalizing it.